docs/sql-ref-syntax-aux-cache-clear-cache.md

```diff
@@ -19,4 +19,16 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Clears all cached RDDs and DataFrames associated with a `SparkSession`.
```

Comment:
@huaxingao Could you double check on the "with a SparkSession" part ? I thought the cached data is scoped beyond a session ?